messaging-activemq/src/main/resources/org/wildfly/extension/messaging/activemq/LocalDescriptions.properties

```diff
@@ -219,8 +222,11 @@ discovery-group.group-address=(Deprecated) Multicast IP address of the group to
 discovery-group.group-port.deprecated=Use instead a socket-binding to specify the discovery group's group port.
 discovery-group.group-port=(Deprecated) UDP port of the multicast group.
 discovery-group.initial-wait-timeout=Period, in ms, to wait for an initial broadcast to give us at least one node in the cluster.
-discovery-group.jgroups-channel=The name used by a JGroups channel to join a cluster.
-discovery-group.jgroups-stack=The name of a stack defined in the org.jboss.as.clustering.jgroups subsystem that is used to form a cluster.
+discovery-group.jgroups-cluster=The logical cluster name.
+discovery-group.jgroups-channel=References the name of a JGroups channel.  If undefined, the default channel will be used.
+discovery-group.jgroups-channel.deprecated=Deprecated. Use jgroups-cluster instead.
+discovery-group.jgroups-stack=References the name of a JGroups channel factory.
+discovery-group.jgroups-stack.deprecated=Deprecated. Use jgroups-channel-ref instead.
```

Comment:
I suppose you mean to use `jgroups-channel` instead

Reply:
Correct. Will fix.